scipy/stats/_hypotests.py

```diff
@@ -1433,3 +1435,242 @@ def cramervonmises_2samp(x, y, method='auto'):
             p = max(0, 1. - _cdf_cvm_inf(tn))
 
     return CramerVonMisesResult(statistic=t, pvalue=p)
+
+
+class TukeyKramerResult:
+    """Result of `scipy.stats.tukeykramer`.
+
+    Attributes
+    ----------
+    sig_level : float
+            The desired significance level (copied from `tukeykramer` input).
+    statistic : float ndarray
+        The computed statistic of the test for each comparison. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
+    pvalue : float ndarray
+        The associated p-value from the studentized range distribution. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
+
+    Methods
+    -------
+    confidence_inteval() :
+        Compute the confidence interval for the significance level
+        specified in the test.
+
+    References
+    ----------
+    .. [1] NIST/SEMATECH e-Handbook of Statistical Methods, "7.4.7.1. Tukey's
+           Method."
+           https://www.itl.nist.gov/div898/handbook/prc/section4/prc471.htm,
+           28 November 2020.
+    """
+
+    def __init__(self, sig_level, statistic, pvalue, _nargs, _nsamples,
+                 _stand_err):
+        self.sig_level = sig_level
+        self.statistic = statistic
+        self.pvalue = pvalue
+        self._nargs = _nargs
+        self._nsamples = _nsamples
+        self._stand_err = _stand_err
+
+    def confidence_interval(self):
+        # determine the critical value of the studentized range using the
+        # appropriate significance level, number of treatments, and degrees
+        # of freedom as determined by the number of data less the number of
+        # treatments. ("Confidence limits for Tukey's method")[1]. Note that
+        # in the cases of unequal sample sizes there will be a criterion for
+        # each group comparison.
+        params = (1 - self.sig_level, self._nargs,
+                  self._nsamples - self._nargs)
+        srd = distributions.studentized_range.ppf(*params)
```

Comment:
```suggestion
        params = (self.sig_level, self._nargs,
                  self._nsamples - self._nargs)
        srd = distributions.studentized_range.isf(*params)
```
No practical difference, but sets a better example, as some distributions have more accurate `isf` methods for small values of the argument.